.github/workflows/deploy.yaml

```diff
@@ -58,22 +58,7 @@ jobs:
       - uses: cowprotocol/autodeploy-action@v1
         if: ${{ github.ref == 'refs/heads/main' }}
         with:
-          pods: "goerli-api-staging,\
```

Comment:
BTW, autodeploy failed for the last 2 commits so it seems like that change (restart pod by image name) was not backwards compatible (which it was supposed to IIRC).

Reply:
Weird, I restarted k8s on Monday and only since Wednesday did deployments start failing (it even looks as if they are still restarting the pods). Will take a look

Reply:
The error is related to us enabling solver/driver colocation and thus autodeploy failing with: 
`[WARNING] from server (NotFound): deployments.apps "xdai-solver-staging" not found`